client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -103,17 +104,18 @@ export const PatternLibrary = ( {
 	const isLoggedIn = useSelector( isUserLoggedIn );
 	const isDevAccount = useSelector( ( state ) => getUserSetting( state, 'is_dev_account' ) );
 
-	const handleViewChange = ( view: 'grid' | 'list' ) => {
-		const currentView = isGridView ? 'grid' : 'list';
+	const type = patternTypeFilter === PatternTypeFilter.REGULAR ? 'pattern' : 'page-layout';
```

Comment:
```suggestion
	const tracksPatternType = patternTypeFilter === PatternTypeFilter.REGULAR ? 'pattern' : 'page-layout';
```
It doesn't have to be this exact name, but seeing as this component already holds a fair amount of code, I think we should be more specific with the variable names.

Reply:
`patternType` sounds okay.